hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -506,4 +506,17 @@ public static long getOMClientRpcTimeOut(Configuration configuration) {
     return OzoneConfiguration.of(configuration)
         .getObject(OMClientConfig.class).getRpcTimeOut();
   }
+
+  /**
+   * Verify key name is a valid name.
+   */
+  public static void validateKeyName(String keyName)
+          throws OMException {
+    try {
+      HddsClientUtils.verifyKeyName(keyName);
+    } catch (IllegalArgumentException e) {
+      throw new OMException("Invalid key name: " + keyName,
```

Comment:
Corresponding to the suggestion for `HddsClientUtils`:

```suggestion
      throw new OMException(e.getMessage(),
```